src/components/shared/ListView/ProductList.tsx

```diff
@@ -0,0 +1,32 @@
+import ProductCard from "@/components/products/ProductCard"
+import {Container} from "@chakra-ui/react"
+import {Products} from "ordercloud-javascript-sdk"
+import {IProduct} from "types/ordercloud/IProduct"
+import ListView from "./ListView"
+import ProductListToolbar from "./ProductListToolbar"
+
+const ProductQueryMap = {
+  s: "Search"
+}
+
+const ProductList = () => {
+  return (
+    <ListView<IProduct>
+      service={Products.List}
+      queryMap={ProductQueryMap}
+      gridOptions={{templateColumns: "repeat(4, 1fr)"}}
```

Comment:
What do you think about opening this up for all chakra styling options? I did something similar for ⁠the [link component](https://github.com/Sitecore/Sitecore.Commerce.Headstart.ReactAdmin/blob/main/src/components/navigation/Link.tsx). You get all of the power of chakra styling, along with the Intellisense